cmake/ReverseDDPostProcess.py

```diff
@@ -0,0 +1,180 @@
+# EnergyPlus, Copyright (c) 1996-2021, The Board of Trustees of the University
```

Comment:
This is the Python post-processing script.  It's purpose is to compare "regular" and "reverse-dd" CSV and MTR files.  There are some complications about these files that make it difficult to compare.